qgroundcontrol.qrc

```diff
@@ -296,18 +296,20 @@
         <file alias="CameraGalleryButton.qml">src/QmlControls/CameraGalleryButton.qml</file>
         <file alias="QGroundControl/Controls/Grid.qml">src/QmlControls/Grid.qml</file>
         <file alias="QGroundControl/Controls/CenterPoint.qml">src/QmlControls/CenterPoint.qml</file>
-        <file alias="CommonTopMenu.qml">src/QmlControls/CommonTopMenu.qml</file>
+        <file alias="QGroundControl/Controls/CommonTopMenu.qml">src/QmlControls/CommonTopMenu.qml</file>
```

Comment:
What is the reason behind this changing path?

Reply:
I set it to the same naming convention as the other files.